src/views/poi_panel.dot

```diff
@@ -8,12 +8,20 @@
        {{= this.panel.renderPartial(this.headerPartial) }}
        {{= this.minimalHourPanel.render() }}
       </div>
+      <div class="poi_panel__content__card__action__container">
         <button class="poi_panel__content__card__action" {{= click(this.showDetail, this) }}>
           <span class="icon-chevrons-right"></span>
           {{= _('SEE MORE', 'poi panel') }}
         </button>
+        {{? this.isDirectionActive }}
+          <button class="poi_panel__content__card__action poi_panel__content__card__action__direction" {{= click(this.openDirection, this) }}>
+            <span class="icon-chevrons-right"></span>
+            {{= _('DIRECTIONS', 'poi panel') }}
+          </button>
+        {{?}}
```

Comment:
These issues have not been corrected:
* wrong icon
* directions button should be the first one

Also, I am not sure about the alignment of these buttons. 
Shouldn't they be put besides the "close" button ?